### DISCUSSION TITLE
Tiny11 (+core) - November '24 release

### DISCUSSION THREAD

**Discussion by ntdevlabs** [ntdevlabs/tiny11builder#279] [Announcements]
This new release adds support for Windows 11 24H2, as well as the various QoL improvements added from various pull requests. 

<hr /><em>This discussion was created from the release <a href='https://github.com/ntdevlabs/tiny11builder/releases/tag/'>Tiny11 (+core) - November '24 release </a>.</em>

**Comment by muhammadbahaa2001** [comment]
> This new release adds support for Windows 11 24H2, as well as the various QoL improvements added from various pull requests. 
> 
> <hr /><em>This discussion was created from the release <a href='https://github.com/ntdevlabs/tiny11builder/releases/tag/'>Tiny11 (+core) - November '24 release </a>.</em>

Adds support for 24H2.
This means it fully supports non-enterprise editions? (aka Home & Pro)

I tried it before this release on non-enterprise editions but it failed in installation.

**Comment by AlexanderHolistic** [comment]
Will this also work with the recent released arm64 version of win11?

**Comment by sl2365** [comment]
I always wanted a slimmed down windows, but never knew how, thanks for this.
I have a few questions though:
1. I assume this requires a standard windows license key?
2. Would my current license key work, as I have a shop bought system, meaning the OS is an OEM version?
3. I would like to use Windows Defender, will that obtain updates to it's AV database?
4. Does everything work as normal, MIDI controllers, USB devices, Class compliance, dot net apps, VisualStudio redistributions apps, etc?
5. What happens should I want to update? I have to wait for you to release the next Tiny11 version? or can I download the latest version of Windows, 25H2 for example, and use the current Tiny11 version? Will it be compatible?
6. Why the hell don't M$ just do this? Ok, no need to answer that, it's rhetorical! :)

Thanks for this! Looking forward to trying it out.

**Comment by Karl-WE** [comment]
hi @ntdevlabs something is confusing with this release, while the diff checker shows another change for the use of WindowsImage, the released version is completely using dism. What happened to 

https://github.com/ntdevlabs/tiny11builder/commit/e8327ffa5ff562ef1f66028f4a68957eda9c67d8

why was a full revert required? The only issue it introduced was with the compression and there were PRs for futher improvement and fixed (not merged yet).

Thank you for a clarification as this took quite an amount of time.
